dbt/adapters/databricks/impl.py

```diff
@@ -256,47 +255,67 @@ def get_relations_without_caching(self, relation: DatabricksRelation) -> Table:
 
         # if there are any table types to be resolved
         if any(not row[3] for row in new_rows):
-            # Get view names and create a dictionay of view name to materialization
+            # Get view names and tables and create a dictionary of name to materialization
+            relation_all_tables = self.Relation.create(
+                database=relation.database,
+                schema=relation.schema,
+                identifier="*"
+            )
             with self._catalog(relation.database):
                 views = self.execute_macro(SHOW_VIEWS_MACRO_NAME, kwargs=kwargs)
+                tables = self.execute_macro(
+                    SHOW_TABLE_EXTENDED_MACRO_NAME, kwargs={"relation": relation_all_tables}
+                )
 
             view_names: Dict[str, bool] = {
                 view["viewName"]: view.get("isMaterialized", False) for view in views
             }
 
+            def parse_type(information: str) -> str:
```

Comment:
The information in this format is a `\n`-delimited list of attributes that are of the form `<attribute_name>: <attribute_value>`.